protocol-units/bridge/contracts/script/DeployAtomicBridgeInitiatorMOVE.s.sol

```diff
@@ -0,0 +1,98 @@
+// To Deploy 
+// forge script AtomicBridgeInitiatorMOVEDeployer --fork-url https://holesky.infura.io/v3/YOUR_INFURA_PROJECT_ID --broadcast --verify --etherscan-api-key YOUR_ETHERSCAN_API_KEY
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.22;
+
+import "forge-std/Script.sol";
+import {AtomicBridgeInitiatorMOVE} from "../src/AtomicBridgeInitiatorMOVE.sol";
+import {TransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract AtomicBridgeInitiatorMOVEDeployer is Script {
+    TransparentUpgradeableProxy public atomicBridgeProxy;
+    TimelockController public timelock;
+    string public atomicBridgeSignature = "initialize(address,address,uint256,uint256)";
+    address public moveAdmin;
+
+    // Parameters
+    address public moveTokenAddress = address(0xC36ba8B8fD9EcbF36288b9B9B0ae9FC3E0645227); 
+    address public ownerAddress = address(0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266); 
+    uint256 public timeLockDuration = 172800; // 48 hours in seconds
+    uint256 public initialPoolBalance = 0 ether; // Initial pool balance
+    uint256 public minDelay = 2 days; // 2-day delay for governance timelock
+
+    // Safe addresses (replace these with actual safe addresses)
+    address public movementLabsSafe = address(0x493516F6dB02c9b7f649E650c5de244646022Aa0); 
```

Comment:
These safes are not on Holesky and there is no Safe on Holesky, if our bridge was deployed, I suggest rerunning and using EOA addresses for upgradeability